.grit/patterns/js/react_hooks.grit

```diff
@@ -427,14 +433,18 @@ pattern first_step($use_ref_from, $handler_callback_suffix) {
         } else {
             $args = .
         },
-
+        if ($use_previous_value_hook <: true) {
+            $previous_value_hook = `\nfunction usePreviousValue(){\n    const ref = useRef();\n    useEffect(() => {\n        ref.current = props;\n    });\n    return ref.current;\n}\n`,
```

Comment:
Snippets can be multi-line.

Reply:
I didn't want to create a noise as this variable is a static code block. 

Let me know what works better in such cases.